[DOP-13016] - add namespace_history table and its api methods (#24)
MobileTeleSystems/horizon: docs/changelog/next_release/24.feature.rst

> Add ``Namespace History``. Now it is possible to view paginated history of actions with selected namespaces.

Reviewer: ```suggestion
Add ``Namespace History``. Now it is possible to view paginated history of actions for specific namespace
```

Мы же не передаем в API список namespace_id, только одно конкретное значение
